src/utils/utils.js

```diff
@@ -11,18 +11,126 @@ const disabilityMap = {
   AUTYZM: "z autyzmem, w tym z zespołem Aspergera"
 };
 
-const calculateValues = values => {
-  const reason = values.applicant.reason;
-  let secondReason = values.applicant.secondReason;
-  if (reason && secondReason) {
-    const firstReason = disabilityMap[reason];
-    secondReason = disabilityMap[secondReason];
-    values.applicant.multipleDisability = [firstReason, secondReason];
-    values.applicant.reason = "SPRZEZONA";
-  }
-  return values;
-};
+class DocumentData {
+  constructor(values) {
+    this.values = values;
+  }
+
+  isMultipleDisability() {
+    return !!(
+      this.values.applicant.reason && this.values.applicant.secondReason
+    );
+  }
+
+  reason() {
+    if (this.isMultipleDisability()) {
+      return "SPRZEZONA";
+    }
+    return this.values.applicant.reason;
+  }
+
+  multipleDisability() {
+    if (!this.isMultipleDisability) {
+      return [];
+    }
+    const firstReasonDescription = disabilityMap[this.values.applicant.reason];
+    const secondReasonDescription =
+      disabilityMap[this.values.applicant.secondReason];
+    return [firstReasonDescription, secondReasonDescription];
+  }
+
+  applicantsList() {
+    const applicants = [
+      {
+        firstName: this.values.applicant.firstName1,
+        lastName: this.values.applicant.lastName1,
+        address: this.values.applicant.address1,
+        postalCode: this.values.applicant.postalCode1
+      }
+    ];
+    if (this.values.applicant.firstName2) {
+      applicants.push({
+        firstName: this.values.applicant.firstName2,
+        lastName: this.values.applicant.lastName2,
+        address: this.values.applicant.address2,
+        postalCode: this.values.applicant.postalCode2
+      });
+    }
+    return applicants;
+  }
+
+  parentsDescription() {
+    const applicant = this.values.applicant;
```

Comment:
Probably you have thought about:
```suggestion
    const {applicant} = this.values;
```
But of course I'm not sure

Reply:
Is this anyhow better to use the destructuring assignment instead of old, javascript direct assignment here?

Reply:
After babel's transpilation, the final form of this should be old, "good" assignment but for being consisted with rest of app I guess new syntax should be used. Unless this is a case when the old syntax is better (but I don't know why.)

Reply:
From my perspective as an outsider, I haven't used to the destructuring assignment syntax to the point that I would use that for assignment of one variable. Although it's not wrong I personally prefer to use simple
```
applicant = this.values.applicant;
```
and I see a real benefit of using the destructuring assignment when want to extract multiple values, like:
```
{ applicant, school } = this.values.applicant;
```

I understand the consistency argument, I was just curious is there any hidden benefit of it that I don't know about.